Include project name in window title

schpet opened the pull request:
## 🧢 Changes

- include the project in the window title (previously it was simply 'GitButler' on all screens)
- update tauri permissions so that can be done from the frontend

## ☕️ Reasoning

makes it easier to differentiate between different gitbutler windows in 
certain tab managers (like [alt-tab](https://alt-tab-macos.netlify.app/))

![Screenshot 2025-09-16 at 21.14.22@2x.png](https://app.gitbutler.com/uploads/39eceea6-2b84-4508-b2d8-63bc4f3b3190)

schpet committed:
Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.

Byron reviewed (commented):
This looks awesome, thanks so much!

Let me reel in @PavelLaptev to get this PR merged!

schpet committed:
add mock for command: plugin:window|set_title

Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.

add mock for command: plugin:window|set_title

Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.

add mock for command: plugin:window|set_title

Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.

add mock for command: plugin:window|set_title

Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.
add mock for command: plugin:window|set_title
add mock for command: plugin:window|set_title
Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.
Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.
Cleanup
Fix the playwright tests by setting window title in a web safe way
Include project name in window title

makes it easier to differentiate between different gitbutler windows in 
certain tab managers. update tauri permissions to allow setting the 
window title.

PavelLaptev commented:
@schpet thanks for the PR!
@Byron LGTM!

schpet commented:
@Byron @PavelLaptev i attempted to fix the failing specs, but i think someone needs to approve the gh actions run before i know if it worked

schpet committed:
Fix the playwright tests by setting window title in a web safe way

schpet committed:
Remove unused import
